<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lapse_window</class>
 <widget class="QMainWindow" name="lapse_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lapse_window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="2" column="1">
       <widget class="QLineEdit" name="images_fld"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Start Time</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Interval (s)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="interval_fld">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="images_btn">
        <property name="text">
         <string>Images</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDateTimeEdit" name="time_fld">
        <property name="displayFormat">
         <string>yyyy-MM-dd h:mm</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="saveas_btn">
        <property name="text">
         <string>Save As</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="saveas_fld"/>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="make_timelapse_btn">
        <property name="text">
         <string>Make Timelapse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
